### ISSUE TITLE
Outdated Docker image

### ISSUE THREAD

**Issue by ghost** [galexrt/docker-staytus#7]
The Docker image was last pushed 8 months ago. In the past 8 months, at least two security issues were fixed in Staytus ([one](https://github.com/adamcooke/staytus/commit/7d360def0ac1ca441438b3ca42f46da0476a5104), [two](https://github.com/adamcooke/staytus/commit/fc4ed22a02731064c30368010e579283e36321be)). Please update the Docker image to include these fixes.

**Comment by galexrt** [comment]
@zul-jin This Docker image uses the `stable` branch of staytus. It does not seem as those two commits are in the `stable` branch (yet).

**Comment by maddprof** [comment]
To piggy back on this subject - have you had any luck getting the image to build from the Master branch? It seems to not want to generate the authie_sessions table no matter I do.

**Comment by galexrt** [comment]
@maddprof Are you talking about having an issue getting a table to generate when building the Docker image from `master`? If yes, have you tried to see if you get the same issue when installing Staytus directly?

I'll probably create a `release-master` branch soon, that will install Staytus from `master`.

**Comment by maddprof** [comment]
So the table doesn't seem to get generated at installation with a docker image.

I can't confirm the version in a standard install right now, I've already killed that virtual machine off.

**Comment by galexrt** [comment]
@maddprof I have pushed a `vmaster` branch which should build from Staytus `master`.
Docker Hub is in progress right now, Quay.io build is already done.

**Comment by galexrt** [comment]
@maddprof Testing the `vmaster` shows that the `authie_sessions` table is not created. Have you already asked at the Staytus issue if that issue is known on `master`?

**Comment by maddprof** [comment]
@galexrt I have not. Do you still have the actual errors that are kicked out to start a thread on it?

**Comment by villasenor** [comment]
It's just that they left the create `authie_sessions` table statement commented out! Was able to get it up and running by just uncommenting the commented out lines in this file: https://github.com/adamcooke/staytus/blob/master/db/migrate/20170608083959_create_authie_sessions.authie.rb

**Comment by galexrt** [comment]
@maddprof Sorry for not responding earlier, the notification email probably got lost in my inbox..

***

From what I can tell the image and the staytus setup is working now.
I have finally added "auto build" for this Docker image now. The auto build will run every month and pull in the latest base image and staytus `master` branch.

***

@villasenor Is uncommenting the mentioned lines required for staytus to fully work?

***

I'm going to close the issue, feel free to reopen if the issue(s) described is still relevant. Thanks!

**Comment by villasenor** [comment]
@galexrt it's required to get the app to start. But once you start going through the Staytus setup wizard, the final page (Stage 3) throws an HTTP 500, with no helpful debug logging in the Docker logs. Authentication to the admin panel also does not work (I assume because stage 3 of the wizard is saving credentials). So further debugging is required.

**Comment by galexrt** [comment]
@villasenor I have just tried to build the image from staytus `master` and I'm getting the same HTTP 500 without any helpful logs.

Is that issue know at the staytus project?

If you wanna try, I have pushed my working staytus `master` build (+ the authie table creation uncomment) here: https://github.com/galexrt/docker-staytus/tree/build_master (`build_master` branch).

***

I'll look into fixing up the image build for `stable` branch and have the monthly build run.
